README: Better description including Client Support Matrix (#58)
KhronosGroup/OpenXR-Inventory: README.md

SPDX-License-Identifier: CC-BY-4.0
-->

> Publicly-maintained data about what runtimes, middleware components or engines (such as Godot, Unity, Unreal, and StereoKit) provide which OpenXR extensions. Together, the Runtime and Client Support Matrices offer a clear overview of extension support across both runtime implementations and client integrations in the OpenXR ecosystem.

Reviewer: Generally LGTM.

Maybe we could remove references to specific engines (which will be information included in the report) and remove the word provide?

```suggestion
Publicly-maintained data about what runtimes and middleware components or engines implement or integrate support for which OpenXR extensions. Together, the Runtime and Client Support Matrices offer a clear overview of extension support across both runtime implementations and client integrations in the OpenXR ecosystem.
```

Author: I like that suggestion. 
I did remove the specific engines and changed "provide" to "support"

Reviewer 2: Looks good to me too, indeed mentioning the game engines seems overkill and will get stale at some point so good to leave out.

Reviewer 3: > I like that suggestion. I did remove the specific engines and changed "provide" to "support"

Your last revision of the text looks great. Concise and clear.

Author: Thanks everyone. Resolving this thread and merging :)
